diagnostics/blocking_neale/doc/blocking_neale.rst

```diff
@@ -0,0 +1,250 @@
+.. This is a comment in RestructuredText format (two periods and a space).
+
+.. Note that all "statements" and "paragraphs" need to be separated by a blank 
+   line. This means the source code can be hard-wrapped to 80 columns for ease 
+   of reading. Multi-line comments or commands like this need to be indented by
+   exactly three spaces.
+
+.. Underline with '='s to set top-level heading: 
+   https://docutils.sourceforge.io/docs/user/rst/quickref.html#section-structure
+
+Rich Neale's Blocking Diagnostic Documentation
+================================
+
+Last update: 09/30/2021
+
+The diagnostic evaluates the blocking frequency in a model by longitude and season
+as determined by the meridional gradient above a threshold value of daily
+500mb height following `D'Andrea et al 1998 <https://doi.org/10.1007/s003820050230>`__.
+
+
+.. Underline with '-'s to make a second-level heading.
+
+Version & Contact info
+----------------------
+
+.. '-' starts items in a bulleted list: 
+   https://docutils.sourceforge.io/docs/user/rst/quickref.html#bullet-lists
+
+Here you should describe who contributed to the diagnostic, and who should be
+contacted for further information:
+
+- Version 2.0, implemented in MDTF v3.0beta3
+- PI: Rich Neale, NCAR, rneale at ucar dot edu
+- Contact: Dani Coleman, NCAR, bundy at ucar dot edu
+
+.. Underline with '^'s to make a third-level heading.
+
+Open source copyright agreement
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+
+
+Functionality
+-------------
+
+
+The annual cycle of blocking frequency is shown for each analyzed case,
+reference ensemble (CESM1) and observational product (ERAI & MERRA)
+requested, in individual Hovmüller figures (longitude vs time) and all
+comparisons are combined in a line plot showing longitude vs blocking
+frequency. Each figure is available for DJF, JJA and ANN.
+
+
+
+
+Required programming language and libraries
+-------------------------------------------
+
+In this section you should summarize the programming languages and third-party 
+libraries used by your diagnostic. You also provide this information in the 
+``settings.jsonc`` file, but here you can give helpful comments to human 
+maintainers (eg, "We need at least version 1.5 of this library because we call
+this function.")
+
+Required model output variables
+-------------------------------
+
+In this section you should describe each variable in the input data your 
+diagnostic uses. You also need to provide this in the ``settings.jsonc`` file, 
+but here you should go into detail on the assumptions your diagnostic makes 
+about the structure of the data.
+
+References
+----------
+
+Here you should cite the journal articles providing the scientific basis for 
+your diagnostic. To keep the documentation format used in version 2.0 of
+the framework, we list references "manually" with the following command:
```

Comment:
Remove the POD developer instructions from the file and complete documentation if needed